src/components/Dashboard/widgets/TokenDetailWidget.tsx

```diff
@@ -1,18 +1,19 @@
-import React, { useEffect, useState } from 'react';
+/** @jsxImportSource @emotion/react */
```

Comment:
What's this for?

Reply:
emotion's css prop won't work otherwise

Reply:
Hmmm the babel plugin should be making this unnecessary.

Reply:
I will investigate.